docs/developer-docs/integrations/sns/launching/launch-summary.md

```diff
@@ -0,0 +1,532 @@
+
+#### Table 3
+
+<table>
+  <tr>
+    <th>Objects in the SNS subnet</th>
+    <th>State</th>
+    <th>Controlled by</th>
+  </tr>
+  <tr>
+    <td>a principal that can deploy to SNS subnet</td>
+    <td class="light-orange-text">revoked because it is 1-time use</td>
+    <td>NA</td>
+  </tr>
+  <tr>
+    <td class="light-green-text">SNS root on the SNS subnet</td>
+    <td class="light-green-text">pre-decentralization swap mode</td>
+    <td class="light-green-text">initial developer neurons</td>
+  </tr>
+  <tr>
+    <td class="light-green-text">initial developer neurons</td>
+    <td class="light-green-text">pre-decentralization swap mode</td>
+    <td class="light-green-text">dapp developer principal</td>
+  </tr>
+  <tr>
+    <td class="light-green-text">treasury account on the SNS Ledger</td>
+    <td class="light-green-text">pre-decentralization swap mode</td>
+    <td class="light-green-text">SNS root</td>
+  </tr>
+  <tr>
+    <td class="light-green-text">swap account on the SNS Ledger</td>
+    <td class="light-green-text">pre-decentralization swap mode</td>
+    <td class="light-green-text">SNS root</td>
+  </tr>
+</table>
+
+### 5. Dapp developers submit an SNS proposal to handover control of their dapp to the SNS.
+  Before the decentralization swap, the developers hand over the control of the dapp to the SNS.
+
+  This includes adding the SNS root canister as the controller of the dapp and removing
+  yourself (and possibly other developers) from the list of
+  controllers. Also, this includes "registering" the dapp with the SNS so that SNS
+  root is aware that it controls these canisters.
+
+  The registration is done by **SNS proposal**.
+  
+  As there are already initial neurons, this,
+  as well as potential upgrades to the dapp, can be realized by SNS proposals and by the
+  majority of initial neurons voting in favor of them.
+  The initial neurons cannot do other things, such as changing
+  the SNS  parameters, as the SNS governance canister is still in
+  pre-decentralization-swap mode.
+
+If successful, at the end of stage, we the following has changed:
+
+#### Table 1
+
+<table>
+  <tr>
+    <th>SNS Process</th>
+    <th>State</th>
+  </tr>
+  <tr>
+    <td>Decentralization swap state</td>
+    <td>Pending</td>
+  </tr>
+   <tr>
+    <td >NNS Proposal #1</td>
+    <td>Approved</td>
+  </tr>
+   <tr>
+    <td>SNS Proposal #1</td>
+    <td class="light-orange-text">Submitted</td>
+  </tr>
+   <tr>
+    <td>NNS Proposal #2</td>
+    <td>Pending</td>
+  </tr>
+</table>
+
+### 6. Proposal #2 (of 3) is passed or rejected
+
+The initial SNS developer neurons are declared in the initial parameters and available at SNS installation.
+
+Since this proposal is passed by the SNS DAO. When one owns SNS neurons or votes on SNS proposals, one is are part of the SNS DAO. It is common for people to vote on both NNS and SNS proposals from within the [NNS Frontend dapp](https://nns.ic0.app), but they are two separate DAOs.
```

Comment:
```suggestion
Since this proposal is passed by the SNS DAO, when one owns SNS neurons or votes on SNS proposals, one is are part of the SNS DAO. It is common for people to vote on both NNS and SNS proposals from within the [NNS Frontend dapp](https://nns.ic0.app), but they are two separate DAOs.
```